apps/condo/domains/miniapp/schema/B2BAppAccessRightSet.test.js

```diff
@@ -975,59 +1048,108 @@ describe('B2BApp permissions for service user', () => {
             canManageTickets: true,
             canReadTicketComments: true,
             canManageTicketComments: true,
+            canReadTicketFiles: true,
+            canManageTicketFiles: true,
+            canReadTicketCommentFiles: true,
+            canManageTicketCommentFiles: true,
         })
         await updateTestB2BAppAccessRight(support, right.id, { accessRightSet: { connect: { id: accessRightSet.id } } })
 
-        // you can update 'canReadTickets', 'canManageTickets', 'canReadTicketComments' and 'canManageTicketComments'
-        await updateTestB2BAppAccessRightSet(support, accessRightSet.id, {
-            canReadTickets: true,
-            canManageTickets: true,
-            canReadTicketComments: true,
-            canManageTicketComments: true,
-        })
-
         await createTestTicket(serviceUser, organization, property)
 
         {
-            const countByUser2 = await Ticket.count(user, { organization: { id: organization.id } })
-            expect(countByUser2).toBe(2)
-
             const countWithPermissions = await Ticket.count(serviceUser, {})
-            expect(countWithPermissions).toBe(2)
-            const contactWithPermissions = await Ticket.getOne(serviceUser, { id: ticket.id })
-            expect(contactWithPermissions.id).toBe(ticket.id)
+            expect(countWithPermissions).toEqual(2)
+
+            const ticketWithPermissions = await Ticket.getOne(serviceUser, { id: ticket.id })
+
+            expect(ticketWithPermissions.id).toEqual(ticket.id)
```

Comment:
I guess checking something like `contact.phone` / `contact.email` and `property.user` might be helpful